__tests__/index.spec.js

```diff
@@ -198,9 +198,7 @@ describe('toMatchImageSnapshot', () => {
 
     const dataArg = runDiffImageToSnapshot.mock.calls[0][0];
     // This is to make the test work on windows
-    ['snapshotsDir', 'diffDir'].forEach((key) => {
-      dataArg[key] = dataArg[key].replace(/\\/g, '/');
-    });
+    dataArg.snapshotsDir = dataArg.snapshotsDir.replace(/\\/g, '/');
```

Comment:
@NarmeenNaveedAhmed is this change meant to be part of this PR?

Reply:
I compared the branch changes with master and reverted everything other than the removal of the greetings workflow action